tests/common/dualtor/dual_tor_mock.py

```diff
@@ -194,7 +194,7 @@ def mock_server_ip_mac_map(rand_selected_dut, tbinfo, ptfadapter, mock_server_ba
 
 
 @pytest.fixture(scope='module')
-def apply_dual_tor_neigh_entries(rand_selected_dut, tbinfo, mock_server_ip_mac_map):
+def apply_dual_tor_neigh_entries(cleanup_mocked_configs, rand_selected_dut, tbinfo, mock_server_ip_mac_map):
```

Comment:
This method cleans up after itself, I think the only place the `cleanup_mocked_configs` fixture is needed is the `apply_active_state_to_orchagent` and `apply_standby_state_to_orchagent` fixtures

Reply:
Yes, those fixtures could cleanup themselves, Making those fixtures depending on `cleanup_mocked_configs` is to ensure that those fixtures could run before `cleanup_mocked_configs` in teardown so that they could find those configs they pushed.

Reply:
I see. In that case, if we are going to config reload during test teardown, then any cleanup performed by these fixtures is redundant - we could just remove everything after the `yield` and change it to `return`.

Reply:
Updated.